src/ontology/mondo-edit.obo

```diff
@@ -441871,84 +441871,69 @@ created_by: http://orcid.org/0000-0001-5208-3432
-is_a: MONDO:0024321 ! disorder of GPI anchor biosynthesis
-property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/4232 xsd:string
-created_by: http://orcid.org/0000-0001-5208-3432
-
-[Term]
-id: MONDO:0100245
-name: acquired paroxysmal nocturnal hemoglobinuria
 def: "Paroxysmal nocturnal hemoglobinuria (PNH) is an acquired clonal hematopoietic stem cell disorder characterized by corpuscular hemolytic anemia, bone marrow failure and frequent thrombotic events." [Orphanet:447]
-comment: Reason: duplicate. This will be merged with MONDO:0100244 paroxysmal nocturnal hemoglobinuria. 'paroxysmal nocturnal hemoglobinuria' appears to always be acquired. We therefore do not need to make the distinction between 'acquired' and 'inherited'.
 subset: gard_rare {source="GARD:0007337"}
-subset: obsoletion_candidate
 subset: ordo_disease {source="Orphanet:447"}
+synonym: "acquired paroxysmal nocturnal hemoglobinuria" EXACT [MONDO:0100245]
+synonym: "hereditary paroxysmal nocturnal hemoglobinuria" EXACT [MONDO:patterns/hereditary]
+synonym: "inherited paroxysmal nocturnal hemoglobinuria" EXACT [MONDO:0100243]
 synonym: "Marchiafava-Micheli disease" EXACT [Orphanet:447]
 synonym: "paroxysmal hemoglobinuria" EXACT [NCIT:C61233]
 synonym: "PNH" EXACT ABBREVIATION [Orphanet:447]
 xref: DOID:0060284 {source="MONDO:equivalentTo"}
 xref: GARD:0007337 {source="MONDO:equivalentTo"}
 xref: HGNC:8957 {source="GARD:0007337"}
+xref: HP:0004818 {source="MONDO:otherHierarchy", source="DOID:0060284"}
 xref: ICD10CM:D59.1 {source="DOID:0060284"}
 xref: ICD10CM:D59.5 {source="ORDO:447/e", source="Orphanet:447", source="ORDO:447/specific", source="DOID:0060284"}
 xref: ICD10CM:D59.6 {source="DOID:0060284"}
 xref: MedDRA:10034042 {source="ORDO:447/e", source="Orphanet:447"}
 xref: MESH:D006457 {source="ORDO:447/e", source="Orphanet:447", source="DOID:0060284"}
 xref: NCIT:C61233 {source="MONDO:equivalentTo", source="DOID:0060284"}
+xref: OMIM:300818 {source="ORDO:447/e", source="Orphanet:447", source="MONDO:superClassOf", source="DOID:0060284"}
+xref: OMIM:615399 {source="Orphanet:447", source="MONDO:superClassOf", source="ORDO:447/btnt", source="DOID:0060284"}
+xref: OMIMPS:300818 {source="MONDO:cjm", source="MONDO:equivalentTo"}
 xref: Orphanet:447 {source="MONDO:equivalentTo", source="DOID:0060284"}
 xref: SCTID:127057004 {source="MONDO:relatedTo", source="DOID:0060284"}
 xref: SCTID:1468004 {source="DOID:0060284"}
 xref: SCTID:154805009 {source="DOID:0060284"}
 xref: SCTID:191223003 {source="DOID:0060284"}
 xref: SCTID:191224009 {source="DOID:0060284"}
 xref: SCTID:191226006 {source="DOID:0060284"}
+xref: SCTID:1963002 {source="MONDO:equivalentTo"}
 xref: SCTID:1963002 {source="MONDO:subClassOf", source="DOID:0060284"}
 xref: SCTID:234377003 {source="DOID:0060284"}
 xref: SCTID:267559009 {source="DOID:0060284"}
-xref: UMLS:C0019050 {source="MONDO:equivalentTo", source="DOID:0060284"}
+xref: UMLS:C0019050 {source="DOID:0060284"}
```

Comment:
I didn't add "MONDO:relatedTo" because this UMLS is not obviously related to this term.